src/sentry/api/endpoints/project_create_sample.py

```diff
@@ -13,7 +13,9 @@ class ProjectCreateSampleEndpoint(ProjectEndpoint):
     permission_classes = (ProjectEventPermission,)
 
     def post(self, request: Request, project) -> Response:
-        event = create_sample_event(project, platform=project.platform, default="javascript")
```

Comment:
does growth team own these endpoints?

Reply:
I don't think we own it although we do use it in quite a few places